lib/pypi-helpers.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+/*
+  Django versions will be specified in the form major.minor
+  trying to sort with `semver.compare` will throw e.g:
+  TypeError: Invalid Version: 1.11
+  because no patch release is specified, so we will define
+  our own functions to parse and sort django versions
+*/
+
+const parseDjangoVersionString = function(str) {
+  if (typeof(str) !== 'string') { return false; }
+  const x = str.split('.');
+  const maj = parseInt(x[0]) || 0;
+  const min = parseInt(x[1]) || 0;
+  return {
+      major: maj,
+      minor: min
+  };
+};
+
+// sort an array of django versions low to high
+const sortDjangoVersions = function(versions) {
+  return versions.sort(function(a, b) {
+    if (parseDjangoVersionString(a).major === parseDjangoVersionString(b).major) {
+      return parseDjangoVersionString(a).minor - parseDjangoVersionString(b).minor;
+    } else {
+      return parseDjangoVersionString(a).major - parseDjangoVersionString(b).major;
+    }
+  });
+};
+
+
+// extract classifiers from a pypi json response based on a regex
+const parseClassifiers = function(parsedData, pattern) {
```

Comment:
Could I ask you to write unit tests for these functions? There are some nice data-driven tests in [lib/lua-rocks-version.spec.js](https://github.com/badges/shields/blob/master/lib/luarocks-version.spec.js), among others.